tests/test_http_parser.py

```diff
@@ -251,6 +251,20 @@ def test_bad_chunked_c(loop: Any, protocol: Any) -> None:
         parser.feed_data(text)
 
 
+def test_empty_header_name_py(loop: Any, protocol: Any) -> None:
+    """Test that empty header names can't be used."""
+    parser = HttpRequestParserPy(
```

Comment:
Plz parametrize this to run against both parsers — C and Py should have feature parity.

Reply:
The C parser allows empty headers in the current release, but will begin disallowing them in the next release. See [this commit](https://github.com/nodejs/llhttp/commit/10ff94eb252e0e7cb792dcde6d40d0e46b794f8a). Until that commit makes it into a release, and the two parsers will exhibit different behavior. I could add a C test and mark it xfail if you'd like?

Reply:
Yes, `xfail` sounds reasonable. And make sure to add a `reason=` string with a link to the information regarding when it'll be okay to drop it. Could also make the xfail marker conditional on the llhttp version.